CMakeLists.txt

```diff
@@ -260,4 +260,14 @@ configure_package_config_file(
     INSTALL_DESTINATION lib/cmake/DDC
 )
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/DDCConfig.cmake DESTINATION lib/cmake/DDC)
+write_basic_package_version_file(
+    ${CMAKE_CURRENT_BINARY_DIR}/DDCConfigVersion.cmake
+    COMPATIBILITY SameMinorVersion
```

Comment:
Add a comment that you follow semantic versioning and that you are currently at version 0.

Reply:
Where should I add that comment ? Near the version declaration ? Isn't it implicit ?

Reply:
I thought to make clear why you use `SameMinorVersion` which is a bit surprising in a general context.

Reply:
Maybe I misunderstood the role of version 0, I thought the API was unstable between minor versions and thus I thought it would make sense to request compatibility between same minor versions ?

Reply:
Yes, you are true. I was just thinking about adding a comment that `SameMinorVersion` is used because you are in the `0.x` versions.

Most projects use `SameMajorVersion` because they are above `0`, so `SameMinorVersion` looks very restrictive when you do have context.

Reply:
I see I agree, I will add the comment, we will remove it as soon as we go to version 1